sdk/avalon_sdk/ethereum/ethereum_worker_registry.py

```diff
@@ -186,9 +186,32 @@ def worker_register(self, worker_id, worker_type, organization_id,
         """
 
         if (self.__contract_instance is not None):
-
+            if not is_valid_hex_str(worker_id):
+                logging.error("Invalid worker id {}".format(worker_id))
+                return None
+            if not isinstance(worker_type, WorkerType):
+                logging.error("Invalid workerType {}".format(worker_type))
+                return None
+            if not is_valid_hex_str(organization_id):
+                logging.error("Invalid organization id {}"
+                              .format(organization_id))
+                return None
```

Comment:
Add function comment that it returns None on error and transaction receipt on success.

Reply:
Done